lib/vanilla-masker.js

```diff
@@ -149,6 +151,10 @@
     }
     masked = masked.replace(clearDelimiter, "");
     masked = masked.length ? masked : "0";
+    var signal = "";
+    if(opts.showSignal === true) {
+      signal = value < 0 || value.startsWith('-') ? "-" : (opts.showPositiveSignal === true ? "+" : "");
```

Comment:
I guess `showPositiveSignal` comes from the original PR (#51), but you did not add it in the options map.
I don't think we want 2 options for this particular case.
Solutions:
- `showSignal` *always* shows the signal, positive or negative
- `showSignal` might be a boolean or a string (`['always', 'positive', 'onlyNegative'].pickOne()`). We keep a single option.
- `showSignal` and `showPositiveSignal` but I'm not a fan. We should also handle the case where `showPositiveSignal` is set, but not `showSignal` (because we're user-friendly, aren't we?)